### DISCUSSION TITLE
请问demo的问题有解决时间的预估么？

### DISCUSSION THREAD

**Discussion by kid101x** [DIYgod/RSSHub#7724] [Q&A] [answered]
最近两天看到订阅报错，然后看了下rsshub.app，不是hosts出错就是“Looks like something went wrong”。

请问有解决时间的预估么？

当然，无论是否能解决，还是非常感谢作者长时间提供demo，并且允许将其订阅。

**Comment by NeverBehave** [comment, marked as answer]
Demo可用性并没有保证，如果不可使用建议不定期尝试，我们保留在未来变更demo的提供方式，目前的想法包括不仅限于：

- 申请临时，定时自销毁实例
- 限定订阅参数/仅提供部分路由展示

同时似乎存在滥用情况，如果环境继续劣化，可能更多限制会继续出台 -- 毕竟严格来说这已经是算是一种爬虫，而仅依托于道德约束可能很难保证大多数使用者的需求

**Reply by kid101x** [reply]
感谢你的回复。

RSSHub项目确实提供了很多个人部署的方法，并且我自己也有一个来处理微博时间线等订阅。

之所以特别关心Demo是否可用，是因为我是一个Inoreader用户，在该平台上有个功能是**文章分数**，它给每篇文章用一个折线箭头体现，猜测是靠用户阅读数来决定的，如果超火的阅读会变成一个火焰图标，实现这些就需要一个条件：
* 订购人数足够多

比如我订阅的微博热搜榜，能看到有大约10000人订阅RSSHub的Demo源，这样，我有时候只要扫一眼热度排序，就可以快速的把热门话题大致了解一下。

当然，这是一个比较小众且个人的需求，如果Demo最终无法解决滥用问题，也可以表示理解。

---
我这两天看了下rsshub.app页面上的debug信息，发现Hot Error Routes前几个似乎都是一些需要高度个人定制的路由，比如现在第一的是`/telegram/channel/:username/:searchQuery?`。

虽然不知道这个和服务器的压力到底是什么关联（因为我其实不懂技术），但是联想到是不是可以出一个机制，就是让Demo不支持所有需要绑定Token类源，以及一些过于细分的源（比如具体到某个bilibili用户，某个微博用户）。

这样是不是可以缓解一些服务器压力？

---
嗯，最后再次感谢每一个开发者的努力。
